srcpkgs/python3-CodeIntel/patches/python3_syntax_error.patch

```diff
@@ -0,0 +1,172 @@
+--- codeintel/ciperf.py	2018-10-15 19:00:15.416225804 +0200
```

Comment:
please add a patch header as described in #2732 . I read the patch and it was not obvious for me what the purpose was. Also is it correct that the target file is .patched?